mods/ra/sequences/infantry.yaml

```diff
@@ -29,6 +26,9 @@ e1:
 		Length: 4
 		Facings: 8
 		Tick: 100
+	liedown:
+		Start: 128
+		Length: 16
 	standup-0:
```

Comment:
I have no idea why I called this `standup-0` back in befe22e170b1a06143f59da9880cbda3a71f5757, but this has been randomly copy pasta-ed to a few more places.  Can you please rename them to `standup` for consistency while we're here.